app/dashboard/views.py

```diff
@@ -1141,6 +1151,188 @@ def get_quickstart_video(request):
+
+    context = {
+        # Header
+        "expiring_bounties_count": d_expiring_bounties_count,
+        "expiring_days_count": d_expiring_days_count,
+        # Stats
+        "submitted_bounties_count": d_submitted_bounties_count,
+        "total_contributors_count": d_total_contributors_count,
+        "total_paid_dollars": usd_format(d_total_paid_dollars),
+        "total_paid_eth": eth_format(d_total_paid_eth),
+        "total_paid_date_since": d_total_paid_date_since,
+        "total_budget_dollars": usd_format(d_total_budget_dollars),
+        "total_budget_eth": eth_format(d_total_budget_eth),
+        "total_budget_use_input_layout": d_total_budget_use_input_layout,
+        "total_budget_used_time_period": d_total_budget_used_time_period,
+        # Payout History
+        "payout_history_weekly": json.dumps(d_payout_history_weekly, ensure_ascii=False, cls=DjangoJSONEncoder),
+        "payout_history_monthly": json.dumps(d_payout_history_monthly, ensure_ascii=False, cls=DjangoJSONEncoder),
+        "payout_history_yearly": json.dumps(d_payout_history_yearly, ensure_ascii=False, cls=DjangoJSONEncoder),
+        # Csv export all bounties this year
+        "csv_all_time_paid_bounties": json.dumps(d_csv_all_time_paid_bounties,
+                                                 ensure_ascii=False,
+                                                 cls=DjangoJSONEncoder),
+        # Tax Reporting
+        "tax_year": d_tax_year,
+        "tax_year_bounties_count": d_tax_year_bounties_count,
+        "tax_year_bounties_worth_dollars": d_tax_year_bounties_worth_dollars,
+        # Latest on your bounties
+        "expired_issues_count": d_expired_issues_count,
+        "expired_issues_worth_dollars": usd_format(d_expired_issues_worth_dollars),
+        "active_bounties_count": d_active_bounties_count,
+        "completed_bounties_count": d_completed_bounties_count,
+        "expired_bounties_count": d_expired_bounties_count,
+        "top_contributors": d_top_contributors,
+        # Table data - outgoing funds and bounties
+        "outgoing_funds_filters": d_outgoing_funds_filters,
+        "outgoing_funds": json.dumps(d_outgoing_funds, ensure_ascii=False, cls=DjangoJSONEncoder),
+        "all_bounties_filters": d_all_bounties_filters,
+        "all_bounties": json.dumps(d_all_bounties, ensure_ascii=False, cls=DjangoJSONEncoder),
+    }
+
+    return TemplateResponse(request, 'funder_dashboard.html', context)
+
+
+@csrf_exempt
+@require_POST
+def update_funder_total_budget(request):
+    data = json.loads(request.body)
+
+    budget_type = None
+    if data['isMonthly']:
+        budget_type = 'monthly'
+    elif data['isQuarterly']:
+        budget_type = 'quarterly'
+
+    budget_usd = data['budget']
+    if budget_type is None or budget_usd is None or float(budget_usd) < 0:
+        return JsonResponse({'status': '400'}, status='400')
+
+    profile = request.user.profile
+
+    profile.funder_total_budget_usdt = budget_usd
```

Comment:
You could update this to use `profile.update(funder_total_budget_usdt=budget_usd, ...)`